bark/models/behavior/behavior_rss/behavior_rss.cpp

```diff
@@ -45,6 +45,11 @@ Trajectory BehaviorRSSConformant::Plan(
     }
   }
 
+  if (last_state_(StateDefinition::TIME_POSITION) < 0) {
```

Comment:
why should the time position ever be smaller than 0? I cannot see that happening